gomigrate.go

```diff
@@ -34,20 +35,34 @@ type Migrator struct {
 	migrations     map[uint64]*Migration
 }
 
+// Logger is the instance of a StdLogger interface to log events to.
+// By default it is set to send all log messages to stdout,
+// but you can set it to redirect wherever you want by instantiating
+// your own logger.
+var Logger StdLogger = log.New(os.Stdout, "[gomigrate] ", log.LstdFlags)
```

Comment:
Let's then set the logger to be the stdout one in `NewMigrator`. We can then allow for a custom logger in a different initializer, like `NewCustomMigrator`.